model_zoo/spleen_ct_segmentation_pytorch/train.py

```diff
@@ -0,0 +1,265 @@
+# Copyright 2020 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import logging
+import os
+import sys
+import tempfile
+from glob import glob
+
+import monai
+import nibabel as nib
+import numpy as np
+import torch
+from monai.data import (
+    CacheDataset,
+    DataLoader,
+    decollate_batch,
+    load_decathlon_datalist,
+    set_track_meta,
+)
+from monai.inferers import sliding_window_inference
+from monai.losses import DiceLoss
+from monai.metrics import DiceMetric
+from monai.networks.layers import Norm
+from monai.networks.nets import UNet
+from monai.transforms import (
+    Activations,
+    AsDiscrete,
+    AsDiscreted,
+    Compose,
+    CropForegroundd,
+    EnsureChannelFirstd,
+    Invertd,
+    LoadImaged,
+    Orientationd,
+    RandCropByPosNegLabeld,
+    SaveImaged,
+    ScaleIntensityRanged,
+    Spacingd,
+)
+from monai.visualize import plot_2d_or_3d_image
+from torch.utils.tensorboard import SummaryWriter
+
+
+def main(tempdir, load_pretrained_ckpt=False):
+    monai.config.print_config()
+    logging.basicConfig(stream=sys.stdout, level=logging.INFO)
+
+    monai.utils.set_determinism(seed=123)
+    torch.backends.cudnn.benchmark = True
+
+    # define path
+    data_file_base_dir = "./data/spleen"
+    data_list_file_path = "./data/dataset_0.json"
+
+    os.makedirs("./models", exist_ok=True)
+    if load_pretrained_ckpt:
+        # transfer learning
+        save_model = "./models/model_transfer.pt"
```

Comment:
Same comment as above, no need to use different file name to save the model weights.

Thanks.

Reply:
Maybe I misunderstood you, but I thought Can was comparing the results of transfer learning with scratch train, thus showing that the transfer learning is more meaningful, so this OPTION should be required?

Reply:
OKOK, I see.
We can keep the logic.

Thanks for the explanation.

Reply:
Thank you Yun for the explanation!